src/Symfony/Component/Form/Command/FormDebugCommand.php

```diff
@@ -83,6 +83,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $io->section('Type extensions');
             $io->listing(array_keys($this->extensions));
         }
+
+        if (count($this->typesFound) > 1) {
+            $io->block('Similar form types found:', null, 'info');
```

Comment:
I think it should be an error, like for ambiguous commands (non 0 exit code) and should look like:

```sh
  Type "FooType" is ambiguous.                                     
  Did you mean one of these?                             
   * Bar\FooType  
   * Baz\FooType     
```